### PR TITLE
[No-issue] refactor: add new Next templates links 

### PR DESCRIPTION
**Pull request by MarianaAguilera**
### Related issue:

[No-issue]

### Changes

Add new Next templates guide links: Guides page and Templates reference.

### Additional links

n/a.

### CODE REVIEW

File: src/content/docs/en/pages/main-menu/additional-resources/templates-and-integrations/understand-azion-templates.mdx [hunk 1]
```diff
@@ -140,6 +156,22 @@ The Azion Next.js Static Boilerplate enables the creation and deployment of a Ne
 
 </Button>
 
+#### React Boilerplate
+
+The Azion React Boilerplate helps you build a React Single-Page Application (SPA) directly on the edge by executing the boilerplate. Several steps are automated, from repository management to edge deployment.
+
+<Button href="/en/documentation/products/guides/react-boilerplate/" text="go to the React Boilerplate guide" variant="secondary"> 
+
+</Button>
```
Comment (bru-andrade) [inline src/content/docs/en/pages/main-menu/additional-resources/templates-and-integrations/understand-azion-templates.mdx:165]:
```suggestion
<Button href="/en/documentation/products/guides/react-boilerplate/" text="go to the React Boilerplate guide" variant="secondary" />
```

File: src/content/docs/en/pages/main-menu/additional-resources/templates-and-integrations/understand-azion-templates.mdx [hunk 2]
```diff
@@ -140,6 +156,22 @@ The Azion Next.js Static Boilerplate enables the creation and deployment of a Ne
...
+
+#### Vue Boilerplate
+
+The Azion Vue Boilerplate provides an automation solution to build a Vue Single-Page Application (SPA) directly on the edge of the network, encapsulating and automating several steps, from repository management to edge deployment.
+
+<Button href="/en/documentation/products/guides/vue-boilerplate/" text="go to the Vue Boilerplate guide" variant="secondary"> 
+
+</Button>
```
Comment (bru-andrade) [inline src/content/docs/en/pages/main-menu/additional-resources/templates-and-integrations/understand-azion-templates.mdx:173]:
```suggestion
<Button href="/en/documentation/products/guides/vue-boilerplate/" text="go to the Vue Boilerplate guide" variant="secondary" />
```

File: src/content/docs/pt-br/pages/menu-principal/recursos-adicionais/templates-e-integracoes/templates-visao-geral.mdx [hunk 3]
```diff
@@ -139,6 +155,22 @@ O **Next.js Static Boilerplate** da Azion permite a criação e implantação de
 
 </Button>
 
+#### React Boilerplate
+
+O **React Boilerplate** da Azion ajuda você a construir uma Single-Page Application (SPA) baseada em React diretamente no edge, executando esse boilerplate. Várias etapas são automatizadas, desde o gerenciamento de repositórios até a implantação.
+
+<Button href="/pt-br/documentacao/produtos/guias/react-boilerplate/" text="consulte o guia do React Boilerplate" variant="secondary">
+
+</Button>
```
Comment (bru-andrade) [inline src/content/docs/pt-br/pages/menu-principal/recursos-adicionais/templates-e-integracoes/templates-visao-geral.mdx:164]:
```suggestion
<Button href="/pt-br/documentacao/produtos/guias/react-boilerplate/" text="consulte o guia do React Boilerplate" variant="secondary" />
```

File: src/content/docs/pt-br/pages/menu-principal/recursos-adicionais/templates-e-integracoes/templates-visao-geral.mdx [hunk 4]
```diff
@@ -139,6 +155,22 @@ O **Next.js Static Boilerplate** da Azion permite a criação e implantação de
...
+
+#### Vue Boilerplate
+
+O **Vue Boilerplate** da Azion fornece uma solução de automação para criar uma Single-Page Application (SPA) do Vue diretamente no edge da rede, encapsulando e automatizando várias etapas, desde o gerenciamento de repositório até a implantação.
+
+<Button href="/pt-br/documentacao/produtos/guias/vue-boilerplate/" text="consulte o guia do Vue Boilerplate" variant="secondary">
+
+</Button>
```
Comment (bru-andrade) [inline src/content/docs/pt-br/pages/menu-principal/recursos-adicionais/templates-e-integracoes/templates-visao-geral.mdx:172]:
```suggestion
<Button href="/pt-br/documentacao/produtos/guias/vue-boilerplate/" text="consulte o guia do Vue Boilerplate" variant="secondary" />
```

**Review by bru-andrade — APPROVED** [review]

File: src/content/docs/pt-br/pages/menu-principal/recursos-adicionais/templates-e-integracoes/templates-visao-geral.mdx [hunk 5]
```diff
@@ -139,6 +151,18 @@ O **Next.js Static Boilerplate** da Azion permite a criação e implantação de
...
 
+#### React Boilerplate
+
+O **React Boilerplate** da Azion ajuda você a construir uma Single-Page Application (SPA) baseada em React diretamente no edge, executando esse boilerplate. Várias etapas são automatizadas, desde o gerenciamento de repositórios até a implantação.
+
+<Button href="/pt-br/documentacao/produtos/guias/react-boilerplate/" text="consulte o guia do React Boilerplate" variant="secondary" />
+
+#### Vue Boilerplate
+
+O **Vue Boilerplate** da Azion fornece uma solução de automação para criar uma Single-Page Application (SPA) do Vue diretamente no edge da rede, encapsulando e automatizando várias etapas, desde o gerenciamento de repositório até a implantação.
```
Comment (hannah-sk) [inline src/content/docs/pt-br/pages/menu-principal/recursos-adicionais/templates-e-integracoes/templates-visao-geral.mdx:162]:
```suggestion
O **Vue Boilerplate** da Azion fornece uma solução de automação para criar uma Single-Page Application (SPA) do Vue diretamente no edge da rede, encapsulando e automatizando várias etapas, desde o gerenciamento de repositórios até a implantação.
```

Comment (hannah-sk) [reply src/content/docs/pt-br/pages/menu-principal/recursos-adicionais/templates-e-integracoes/templates-visao-geral.mdx:162]: está assim no de cima. desconsidera se for diferente, mesmo

**Review by hannah-sk — APPROVED** [review]

**Pull request merged**